sockets/challonge.js

```diff
@@ -18,9 +20,15 @@ module.exports = function(io) {
         socket.emit('update challonge', challongeData);
         connectedSockets++;
 
+        // if we go from 0 to 1 socket, start polling again
+        pollChallonge();
+
         socket.on('disconnect', function() {
             console.log('Challonge disconnected');
             connectedSockets--;
+            if (connectedSockets === 0) {
+                clearTimeout(timeout);
+            }
```

Comment:
@Camtendo you made some comment here and deleted it or something. It was a good point, so I'm going to fix it. haha